src/javascript/RichTextCKEditor5/RichTextCKEditor5.scss

```diff
@@ -4,12 +4,6 @@
   max-height: 40vh;
 }
 
-.toolbar {
```

Comment:
Have you tried with very long text so that the toolbar stays visible?
Might have been because of changes to the Classic Editor